### ISSUE TITLE
Feature request: Mouse snapback for gyro.

### ISSUE THREAD

**Issue by nickajeglin** [Ryochan7/DS4Windows#2186]
Hello, I have an application where I need somethin to happen when I _release_ a trigger.

I use a soft trigger pull to enable gyro mouse in elite dangerous to let me yaw. When I let go of the trigger, the mouse stays where I left it, as expected. 

I would love an option that would allow the mouse to snap back to center, either when the trigger is released, or when the gyro is disabled. The touchpad "absolute" mouse mode does exactly that, so it would be amazing to have that option for the gyro as well.

**Comment by BitGrub** [comment]
Yeah I'm looking through the options now and I'm surprised there's no "center cursor" function. Would be very useful for motion controls users

**Comment by Ryochan7** [comment]
Not necessarily. It depends on the game. A lot of games can exhibit some extremely weird behavior if it detects absolute mouse events. If you have tested it with the Touchpad then at least it would be technically possible with Elite Dangerous.

Absolute mouse support in DS4Windows is almost non-existent. Hardly ever have a need for it and there have been some old issues I have encountered that I have never solved. Need to solve some issues regarding defining the bounding box for the cursor outside of just mapping to the whole screen.

**Comment by BitGrub** [comment]
That's annoying, but I'd still use it just browsing the internet from the couch. The game _Dream_'s fully motion-controllable UI has spoiled me

**Comment by nickajeglin** [comment]
I figured it's a pretty niche use case. E:D has mouse steering, so I have it set to enable gyro aim on a soft pull of the right trigger. That way I can get all 6 degrees of freedom by having yaw on the gyro when I need it. Without some kind of re-centering, you just keep spinning the same direction even after you disable the gyro by releasing the trigger. 

Luckily E:D does have a feature that auto re-centers the the mouse, but it makes the control feel soft because it pulls against your input the whole time and then only returns to center slowly. Instant snapback would be better, as I could stop the rotation instantly just by releasing the trigger. 

The other option would be to use elite's built in "center mouse" keybind, but that's unwieldly unless there's a way to fire the key press when the trigger releases. I know that macros are pretty basic right now, but do you know if there is a way to do something conditionally? Like wait while(trigger down), then fire a keybind after it comes back up?

In any case, it's certainly usable as is, this would just be a nice-to-have.
